lib/services/response.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:wildlife_api_connection/api_client.dart';
+import 'package:wildlife_api_connection/response_api.dart';
+
+class ResponseService {
```

Comment:
Make use of config for maintainability and prevent debugging sessions. Developers expect to change the baseurl using .env files.

Suggested change:
  final _responselApi = ReponseApi(
    AppConfig.shared.apiClient,
  );

Maybe package needs to be updated? why does baseurl include '/interaction'?